python/ndpi_typestruct.py

```diff
@@ -229,7 +229,6 @@ class CustomCategories(Structure):
     ("detection_bitmask", NDPIProtocolBitMask),
     ("generic_http_packet_bitmask", NDPIProtocolBitMask),
```

Comment:
```suggestion
```

Reply:
Thanks for spotting that! Python bindings are a completely mess...
Fixed

Reply:
True. We might need some sort of automation there.